bin/postman-test/collections/core-data.postman_collection.json

```diff
@@ -1849,7 +1849,7 @@
 						],
 						"body": {
 							"mode": "raw",
-							"raw": "{\n\t\"origin\":1471806366619,\n\t\"device\":\"livingroomthermostat\",\n\t\"name\":\"co2\",\n\t\"value\":\"12\"\n}"
+							"raw": "{\n\t\"origin\":1471806366619,\n\t\"device\":\"livingroomthermostat\",\n\t\"name\":\"co2\",\n\t\"value\":\"12.2\",\n\t\"valueType\":\"Float32\",\n\t\"mediaType\":\"foo\",\n\t\"floatEncoding\": \"ieee23.2\"\n}"
```

Comment:
In this case, it seems mediaType is unnecessary for float type?

Reply:
Good catch, thanks